test/conformance/source/environment.cpp

```diff
@@ -197,7 +197,8 @@ KernelsEnvironment *KernelsEnvironment::instance = nullptr;
 KernelsEnvironment::KernelsEnvironment(int argc, char **argv,
                                        std::string kernels_default_dir)
     : DevicesEnvironment(argc, argv),
-      kernel_options(parseKernelOptions(argc, argv, kernels_default_dir)) {
+      kernel_options(
+          parseKernelOptions(argc, argv, std::move(kernels_default_dir))) {
```

Comment:
nit: What coverity issue does it fix exactly? I think a better fix would be to change the parameter type to `const std::string & kernels_default_dir` - it shows the intention better

Reply:
It is a ```copy_instead_of_move``` issue. Done.